bundle/Resources/views/content_view_fallback/to_ez_platform.html.twig

```diff
@@ -0,0 +1,13 @@
+{% if ezpublish.configResolver.getParameter('ng_fallback_with_subrequest') == true %}
+    {{ render(
+        controller(
+            'ez_content:viewAction', {
+                'content': content.innerContent,
+                'location': location.innerLocation,
+                'viewType': view_type
+            }
+        )
+    ) }}
+{% else %}
+    {{ ng_ez_view_content(location|default(content), view_type) }}
```

Comment:
content.innerContent & location.innerLocation ?

Reply:
`ng_ez_view_content` also handles Site API Content and Location, but using inner objects would be more clear, will update.

Reply:
Updated in https://github.com/netgen/ezplatform-site-api/pull/152/commits/66c10861232d12fd8ea4d8517a2e7bea6a980081.